### ISSUE TITLE
DX Tool Failed

### ISSUE THREAD

**Issue by mrfrasier** [BowlerHatLLC/vscode-as3mxml#395]
Hola,

Having an issue building my project for Android. I can build for iOS no problem, but any time I run "Package Android [debug|release]" I get:

> Adobe AIR code signing password:
> dx tool failed:
> The terminal process terminated with exit code: 17

Right after entering my password.

I know the password is right as entering a wrong password (or invalid path to a .p12 file) will throw an error as such.

I am using AIR 31 with a .p12 file that works fine if I try to build the same project via FLA in Adobe Animate. JDK version is 1.8.0_111

I've googled a bit and see some issues from years past with older 32bit certs, but this is def a 64 bit cert, and as I mentioned, it works fine in Animate. Sadly, as it just says failed with no text after, I don't have much to go on.

Any ideas?

**Comment by joshtynjala** [comment]
This thread seems to indicate that using a newer Java 1.8 update may help:

https://forums.adobe.com/thread/2281743

Also, if you have multiple Java versions installed, it may be a good idea to try the setting that tells vscode-as3mxml to use a specific Java path (just to be sure that it isn't trying to use an unexpected version).

**Comment by joshtynjala** [comment]
That same thread mentions that the 32-bit and 64-bit builds of the same JVM may behave differently, so it might be worth noting whether your Java version is 64-bit or not.

**Comment by mrfrasier** [comment]
Thanks Josh,

Neglected to mention I did try setting the java path in my settings for the workspace to what AIR should be using.

That adobe thread is similar to the others I found which seems to point to the java version, but the odd thing is that it works fine in animate.


Thought I’d post in case I was kissing something obvious, but will continue to tinker and share if I find a solution.

Thanks!
